### ISSUE TITLE
Add simple power ups..can we do it?!

### ISSUE THREAD

**Issue by dsantorojr** [arminm/multegula#167]
I'm just thinking changing paddle sizes, adding a life, and possibly doing "momentum"... These would require very little extra UI code.

**Comment by dsantorojr** [comment]
tagging @garrettmiller here...you'd have to update the level files so the power ups are deterministic and the colors match certain power ups.

**Comment by garrettmiller** [comment]
Acknowledged.

**Comment by dsantorojr** [comment]
No. No we can't do it. This is going to stay firmly in the optional column.
